### DISCUSSION TITLE
Missing ImageMagick-7.1.0-36.tar.xz when do install via homebrew

### DISCUSSION THREAD

**Discussion by fat-man-forever** [tlk/homebrew-imagemagick-x11#7] [Q&A] [answered]
when i tried
`brew install tlk/imagemagick-x11/imagemagick`
I got:

> Error: imagemagick: Failed to download resource "imagemagick"
> Download failed: https://www.imagemagick.org/download/releases/ImageMagick-7.1.0-36.tar.xz

It seems the file 'ImageMagick-7.1.0-36.tar.xz' is no longer there. What should I do now

**Comment by tlk** [comment, marked as answer]
Thanks for the heads-up. I have updated the patch-file and this should solve the issue. You can run `brew update` and try again.
